build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
yes, `rebar3 compile` on a fresh clone works, but not after a `rebar compile` or after `make`, after a clean. Although that use case is admittedly a bit odd, if it is not too much work, it'd be nice if it could work.